### ISSUE TITLE
Conflict between PyFluent 0.12.dev10 and Google Protobuf 3.20.0.

### ISSUE THREAD

**Issue by iazehaf** [pyansys/pyfluent#1277]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Runtime error when running a python script where ``ansys.fluent.core`` and ``google.protobuf.struct_pb2`` are imported. 

`
Traceback (most recent call last):
  File ".\test_case.py", line 4, in <module>
    import ansys.fluent.core
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\ansys\fluent\core\__init__.py", line 9, in <module>
    from ansys.fluent.core.launcher.launcher import (  # noqa: F401
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\ansys\fluent\core\launcher\launcher.py", line 17, in <module>
    from ansys.fluent.core.fluent_connection import _FluentConnection
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\ansys\fluent\core\fluent_connection.py", line 16, in <module>
    from ansys.fluent.core.services.datamodel_tui import (
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\ansys\fluent\core\services\datamodel_tui.py", line 10, in <module>
    from ansys.api.fluent.v0 import datamodel_tui_pb2 as DataModelProtoModule
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\ansys\api\fluent\v0\datamodel_tui_pb2.py", line 16, in <module>
    from ansys.api.fluent.v0.google.protobuf import struct_pb2 as ansys_dot_api_dot_fluent_dot_v0_dot_google_dot_protobuf_dot_struct__pb2
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\ansys\api\fluent\v0\google\protobuf\struct_pb2.py", line 23, in <module>
    serialized_pb=_b('\n0ansys/api/fluent/v0/google/protobuf/struct.proto\x12\x0fgoogle.protobuf\"\x84\x01\n\x06Struct\x12\x33\n\x06\x66ields\x18\x01 \x03(\x0b\x32#.google.protobuf.Struct.FieldsEntry\x1a\x45\n\x0b\x46ieldsEntry\x12\x0b\n\x03key\x18\x01 \x01(\t\x12%\n\x05value\x18\x02 \x01(\x0b\x32\x16.google.protobuf.Value:\x02\x38\x01\"\xea\x01\n\x05Value\x12\x30\n\nnull_value\x18\x01 \x01(\x0e\x32\x1a.google.protobuf.NullValueH\x00\x12\x16\n\x0cnumber_value\x18\x02 \x01(\x01H\x00\x12\x16\n\x0cstring_value\x18\x03 \x01(\tH\x00\x12\x14\n\nbool_value\x18\x04 \x01(\x08H\x00\x12/\n\x0cstruct_value\x18\x05 \x01(\x0b\x32\x17.google.protobuf.StructH\x00\x12\x30\n\nlist_value\x18\x06 \x01(\x0b\x32\x1a.google.protobuf.ListValueH\x00\x42\x06\n\x04kind\"3\n\tListValue\x12&\n\x06values\x18\x01 \x03(\x0b\x32\x16.google.protobuf.Value*\x1b\n\tNullValue\x12\x0e\n\nNULL_VALUE\x10\x00\x42\x81\x01\n\x13\x63om.google.protobufB\x0bStructProtoP\x01Z1github.com/golang/protobuf/ptypes/struct;structpb\xf8\x01\x01\xa2\x02\x03GPB\xaa\x02\x1eGoogle.Protobuf.WellKnownTypesb\x06proto3')
  File "D:\AnsysDev\00.Tests\protobuf-error-test-case\.venv\lib\site-packages\google\protobuf\descriptor.py", line 1024, in __new__
    return _message.default_pool.AddSerializedFile(serialized_pb)
TypeError: Couldn't build proto file into descriptor pool!
Invalid proto descriptor for file "ansys/api/fluent/v0/google/protobuf/struct.proto":
  google.protobuf.Struct.fields: "google.protobuf.Struct.fields" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Struct.FieldsEntry.key: "google.protobuf.Struct.FieldsEntry.key" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Struct.FieldsEntry.value: "google.protobuf.Struct.FieldsEntry.value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Struct.FieldsEntry: "google.protobuf.Struct.FieldsEntry" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Struct: "google.protobuf.Struct" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.kind: "google.protobuf.Value.kind" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.null_value: "google.protobuf.Value.null_value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.number_value: "google.protobuf.Value.number_value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.string_value: "google.protobuf.Value.string_value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.bool_value: "google.protobuf.Value.bool_value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.struct_value: "google.protobuf.Value.struct_value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value.list_value: "google.protobuf.Value.list_value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Value: "google.protobuf.Value" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.ListValue.values: "google.protobuf.ListValue.values" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.ListValue: "google.protobuf.ListValue" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.NULL_VALUE: "google.protobuf.NULL_VALUE" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.NULL_VALUE: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "NULL_VALUE" must be unique within "google.protobuf", not just within "NullValue".
  google.protobuf.NullValue: "google.protobuf.NullValue" is already defined in file "google/protobuf/struct.proto".
  google.protobuf.Struct.FieldsEntry.value: "google.protobuf.Value" seems to be defined in "google/protobuf/struct.proto", which is not imported by "ansys/api/fluent/v0/google/protobuf/struct.proto".  To use it here, please add the necessary import.
  google.protobuf.Struct.fields: "google.protobuf.Struct.FieldsEntry" seems to be defined in "google/protobuf/struct.proto", which is not imported by "ansys/api/fluent/v0/google/protobuf/struct.proto".  To use it here, please add the necessary import.
  google.protobuf.Value.null_value: "google.protobuf.NullValue" seems to be defined in "google/protobuf/struct.proto", which is not imported by "ansys/api/fluent/v0/google/protobuf/struct.proto".  To use it here, please add the necessary import.
  google.protobuf.Value.struct_value: "google.protobuf.Struct" seems to be defined in "google/protobuf/struct.proto", which is not imported by "ansys/api/fluent/v0/google/protobuf/struct.proto".  To use it here, please add the necessary import.
  google.protobuf.Value.list_value: "google.protobuf.ListValue" seems to be defined in "google/protobuf/struct.proto", which is not imported by "ansys/api/fluent/v0/google/protobuf/struct.proto".  To use it here, please add the necessary import.
  google.protobuf.ListValue.values: "google.protobuf.Value" seems to be defined in "google/protobuf/struct.proto", which is not imported by "ansys/api/fluent/v0/google/protobuf/struct.proto".  To use it here, please add the necessary import.
`

Another way to reproduce the bug is to install ansys-discovery and to import both ``ansys.discovery.core`` and ``ansys.fluent.core``.

### 📝 Steps to reproduce

- Create a virtual environment: `python -m venv .venv`
- Activate the virtual environment: `.venv/Scripts/Activate.ps1`
- Install pyfluent: `pip install ansys-fluent-core==0.12.dev10`
- Create a python script and add the following imports: `import google.protobuf.struct_pb2` and `import ansys.fluent.core `
- Run the script: `python script.py`

### 💻 Which operating system are you using?

Windows

### 🐍 Which Python version are you using?

3.7

### 📦 Installed packages

```shell
ansys-api-discovery                   0.3.0      
ansys-api-fluent                      0.3.5      
ansys-api-platform-instancemanagement 1.0.0b3    
ansys-discovery                       0.2.1      
ansys-discovery-server-addin          0.0.30.dev0
ansys-fluent-core                     0.12.dev10 
ansys-platform-instancemanagement     1.0.2      
appdirs                               1.4.4      
cachetools                            5.2.1      
certifi                               2022.12.7  
charset-normalizer                    3.0.1      
google-api-core                       2.11.0     
google-api-python-client              2.72.0     
google-auth                           2.16.0     
google-auth-httplib2                  0.1.0      
googleapis-common-protos              1.58.0     
grpcio                                1.51.1     
h5py                                  3.7.0      
httplib2                              0.21.0     
idna                                  3.4        
importlib-metadata                    6.0.0      
numpy                                 1.21.6     
pandas                                1.3.5      
pip                                   20.1.1     
protobuf                              3.20.0     
protoc-gen-swagger                    0.1.0      
pyasn1                                0.4.8      
pyasn1-modules                        0.2.8      
pyparsing                             3.0.9      
python-dateutil                       2.8.2      
pytz                                  2022.7.1   
requests                              2.28.2     
rsa                                   4.9        
setuptools                            47.1.0     
six                                   1.16.0     
typing-extensions                     4.4.0      
uritemplate                           4.1.1      
urllib3                               1.26.14    
zipp                                  3.11.0
```

**Comment by mkundu1** [comment]
Issue is resolved in the main branch, will be available in the next dev package

**Comment by iazehaf** [comment]
Hi @mkundu1,
Thanks. Looking forward to test it.

**Comment by mkundu1** [comment]
@iazehaf This is fixed in v0.13.dev2 (install with `pip install ansys-fluent-core==0.13.dev2`)

**Comment by iazehaf** [comment]
@mkundu1 thank you very much. Tested. Working.
